tests/feature_creation_tests.py

```diff
@@ -1,9 +1,15 @@
 import pandas as pd
-
+import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '../')))
+print(sys.path)
+from features.taped_stats import TapedStats
 class FeatureCreationTests():
     def __init__(self) -> None:
         self.fights_df = pd.read_csv('tests/test_fights.csv', encoding='latin-1')
         self.fighter_df = pd.read_csv('tests/test_fighters.csv', encoding='latin-1')
+        self.taped_stats = TapedStats()
+        self.expected_df = pd.read_csv('tests/fights_with_empty_expected_values.csv', encoding='latin-1')  # The CSV with filled expected values
```

Comment:
The name is a little confusing, shouldn't it be something like _fights_with_expected_taped_stats_values.csv_?